extensions/todoist/src/components/TaskListItem.tsx

```diff
@@ -81,7 +80,7 @@ export default function TaskListItem({
   }
 
   if (task.deadline?.date) {
-    const text = displayDate(task.deadline.date);
+    const text = DisplayDateTime(task.deadline.date);
```

Comment:
Deadlines are always formatted as full-day dates, so this may not be useful here.